services/workflows-service/scripts/seed.ts

```diff
@@ -92,12 +94,14 @@ async function seed(bcryptSalt: string | number) {
     '1',
     env.API_KEY,
     'https://assets-global.website-files.com/62827cf4fe5eb528708511d4/645511cb3d3dd84ee28fe04d_CyberAgent.svg',
+    `webhook-shared-secret-${env.API_KEY}`,
   );
   const customer2 = await createCustomer(
     client,
     '2',
     `${env.API_KEY}2`,
     'https://assets-global.website-files.com/62827cf4fe5eb528708511d4/645d26f285bd18467470e7cd_zenhub-logo.svg',
+    `webhook-shared-secret-${env.API_KEY}2`,
```

Comment:
Can the customer set a general webhook address?

Reply:
WDYM? Can you explain please?